.github/workflows/oracle-workflow-test.yml

```diff
@@ -25,28 +24,52 @@ jobs:
     steps:
       - name: Checkout Action
         uses: actions/checkout@v2
+
       - name: Get Date
         id: get-date
         run: |
           echo "::set-output name=date::$(/bin/date -u "+%Y-%m")"
         shell: bash
```

Comment:
I actually intentionally kept that the same for now.  

My thought process is that this job runs very selectively and there isn't much in terms of POM changes that would have an impact on this connector.  If Quarkus changes, no impact.  So I felt that re-downloading a half-dozen artifacts for this job as they change over a 30 day period was faster than re-downloading 500mb worth of artifacts that takes 10-15m.

Since this job basically would require a change to `debezium-core` or `debezium-connector-oracle` to be triggered, and runs on pushes only, I left it as-is.

If we want to apply the POM hash to this just understand that it will increase push runs by at least 10-15 minutes even if a dependency changed that has no impact on Oracle at all.